crates/rome_formatter/src/builders.rs

```diff
@@ -779,9 +782,52 @@ impl<Context> std::fmt::Debug for Dedent<'_, Context> {
 /// );
 /// ```
 ///
-/// You can see that the printer indents the function's `}` by two spaces as specified by the IR and not with a tab.
-/// You can further see that alignments behave the same as an `indent` if you nest `indent` inside an align.
+/// You can see that:
+///
+/// * the printer indents the function's `}` by two spaces as specified by the IR and not with a tab.
+/// * that `align` acts the same as an `indent` if you nest `indent` inside an align.
```

Comment:
I find this phrase difficult to understand. Should we specify what "acts the same as" means here? Because I don't get it. Also, I don't understand the nesting part. It seems that I was supposed to know something about it but the documentation doesn't mention it 

```suggestion
/// * that `align` acts the same as an `indent` if you nest `indent` inside an `align`.
```

Reply:
I tried to rephrase the explanation. I hope that helps